src/components/updatemenus/attributes.js

```diff
@@ -11,6 +11,7 @@
 var fontAttrs = require('../../plots/font_attributes');
 var colorAttrs = require('../color/attributes');
 var extendFlat = require('../../lib/extend').extendFlat;
+var padAttrs = require('../../plots/pad_attributes');
```

Comment:
> Font attributes would probably go with it?

yep, exactly.



> so can try to cut down on plots/ bit by bit (pun?)

No action required in this PR. I just wanted to mention something I've been thinking about for a while now.